### PR TITLE
feat: move UpdateTagValue feature to ResourceManager

### PR DESCRIPTION
**Pull request by lutengda**
# Required checklist
- [ ] Sample config files updated (`config`,`meta/config` and `default config`) 
- [ ] If there are user-facing changes, the documentation needs to be updated prior to approving the PR( [Link]() )
- [ ] If there are any breaking changes to public APIs, please add the `api change` label.
- [ ] Signed [CLA](https://cla-assistant.io/cnosdb/cnosdb) (if not already signed)

# Which issue does this PR close?

[//]: # (We generally require a GitHub issue to be filed for all bug fixes and enhancements and this helps us generate change logs for our releases. You can link an issue to this PR using the GitHub syntax. For -- example `Closes #123` indicates that this PR will close issue #123.)

Related #.

# Rationale for this change

[//]: # (Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
 Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.)

# Are there any user-facing changes?

[//]: # (There is no need to duplicate the description in the issue here, but it is sometimes worth providing a summary of the individual changes in this PR.)

### CODE REVIEW

File: coordinator/src/resource_manager.rs [hunk 1]
```diff
@@ -286,7 +290,6 @@ impl ResourceManager {
         let names = resourceinfo.get_names();
         if names.len() == 4 {
```
Comment (yukkit) [inline coordinator/src/resource_manager.rs:291]: Try to avoid this kind of logic without clear semantics

Comment (lutengda) [reply coordinator/src/resource_manager.rs:291]: optimizations will be made later

File: common/models/src/schema.rs [hunk 2]
```diff
@@ -56,41 +57,46 @@ pub const USAGE_SCHEMA: &str = "usage_schema";
...
-    UpdateTagValue,
+    DropTenant(String),
+    DropDatabase(String, String),
+    DropTable(String, String, String),
+    AddColumn(String, TskvTableSchema, TableColumn),
+    DropColumn(String, String, TskvTableSchema),
+    AlterColumn(String, String, TskvTableSchema, TableColumn),
+    RenameTagName(String, String, String, TskvTableSchema),
+    UpdateTagValue(
+        String,
```
Comment (yukkit) [inline common/models/src/schema.rs:70]: extract Struct to make each field have a clear meaning

File: coordinator/src/service.rs [hunk 3]
```diff
@@ -1080,39 +1106,48 @@ impl Coordinator for CoordService {
...
+            .collect();
+
+        let resourceinfo = ResourceInfo::new(
+            0,
+            vec![
+                tenant.to_string(),
+                db.to_string(),
+                table_name.to_string(),
+                "UpdateTagsValue-".to_owned() + &table_schema.schema_id.to_string(),
+            ],
```
Comment (yukkit) [inline coordinator/src/service.rs:1140]: Refactor the name of resource info, which can be done later.

**Review by yukkit — APPROVED** [review]

**Pull request merged**